src/components/Integrations/Shopify/Shopify.js

```diff
@@ -39,6 +41,33 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
       color: #33ad73;
     }
   `;
+
+  const Table = ({ list }) => (
+    <table className="dp-c-table">
+      <thead>
+        <tr>
+          <th>{_('shopify.table_list')} </th>
+          <th> {_('shopify.table_subscribers_count')}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {list.state === SubscriberListState.ready ? (
+          <tr>
+            <td>{list.name}</td>
+            <td>{list.amountSubscribers}</td>
+          </tr>
+        ) : (
+          <tr className="sync">
+            <td>{list.name}</td>
+            <td className="text-sync">
+              <span className="ms-icon icon-clock"></span>
+              Synchronizing...
```

Comment:
This is new, right? what do you think about adding a test?

Reply:
Also I've missed the resourced text here. Sure a test can help.